Sources/TuistKit/Commands/InitCommand.swift

```diff
@@ -327,6 +322,24 @@ class InitCommand: NSObject, Command {
                                          content: PlaygroundGenerator.defaultContent())
     }
 
+    fileprivate func generateStoryboards(name: String, path: AbsolutePath, platform: Platform, product: Product) throws {
+        let sourcesPath = path.appending(component: "Sources")
+
+        if product == .app, platform.supportsLaunchScreen {
+            try storyboardGenerator.generate(path: sourcesPath,
+                                             name: "Launch Screen",
```

Comment:
I wouldn't pass a name. Since we are generating a project, we can stick to a convention that developers can change later on.

Reply:
Removed name until I have a solution to get the name from `Project.swift`